app/src/androidTest/java/fr/free/nrw/commons/SettingsActivityTest.java

```diff
@@ -15,15 +11,18 @@
 import android.support.test.runner.AndroidJUnit4;
 import android.view.View;
 
-import fr.free.nrw.commons.settings.SettingsActivity;
-
-import java.util.Map;
-
 import org.hamcrest.Matcher;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
+import java.util.Map;
+
+import fr.free.nrw.commons.settings.SettingsActivity;
+
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.anything;
+
```

Comment:
Sorry about this unrelated change - Android Studio automatically reordered them without me noticing - can revert if needs be